apps/create_rtdfiles/4C_create_rtdfiles_utils.cpp

```diff
@@ -155,21 +155,21 @@ namespace RTD
   /*----------------------------------------------------------------------*/
   void write_paragraph(std::ostream& stream, std::string paragraph, size_t indent)
   {
-    size_t mathstartpos = paragraph.find("\f$");
+    size_t mathstartpos = paragraph.find("$");
```

Comment:
@ischeider Not directly related to this PR but since you modify the rtd code: We are ready to generate the automatic part of the docs from `4C_metadata.yaml`. It contains all the information we have about all parts of the input file. I suppose a couple of python scripts are a lot easier to maintain, read and write. C++ is overall not a great fit for this kind of file manipulation jobs.